server/services/TrustService.js

```diff
@@ -91,38 +93,112 @@ class TrustService {
       originatorWallet,
     });
 
+    // the log should show up on both requester and requestee
+    await this._event.logEvent({
+      loggedInWalletId: requesterWalletDetails.id,
+      type: 'trust_request',
+      payload: {
+        requesteeWallet: requesteeWalletDetails.name,
+        requesterWallet: requesterWalletDetails.name,
+        trustRequestType,
+      },
+    });
+
+    // the log should show up on both requester and requestee
+    await this._event.logEvent({
+      loggedInWalletId: requesteeWalletDetails.id,
+      type: 'trust_request',
+      payload: {
+        requesteeWallet: requesteeWalletDetails.name,
+        requesterWallet: requesterWalletDetails.name,
+        trustRequestType,
+      },
+    });
+
     return trustRelationship;
   }
 
   async acceptTrustRequestSentToMe({ walletLoginId, trustRelationshipId }) {
+    const trustRelationship = await this._trust.getTrustRelationshipsById(
+      trustRelationshipId,
+    );
+
+    // the log should show up on both requester and requestee
+    await this._event.logEvent({
```

Comment:
these calls should be put in a transaction

Reply:
Will include all the logEvent functions within the knex transactions!